src/cdk/scrolling/virtual-for-of.ts

```diff
@@ -240,7 +240,7 @@ export class CdkVirtualForOf<T> implements CollectionViewer, DoCheck, OnDestroy
   ngOnDestroy() {
     this._viewport.detach();
 
-    this._dataSourceChanges.next();
+    this._dataSourceChanges.next(null!);
```

Comment:
Could you add a comment that explains why this is here?

Reply:
This seems a little bit dangerous, since the code on the other end is expecting it not to be null

Reply:
How about `undefined!`?

Reply:
Oh I see, previously this was undefined anyways, yeah that should be fine then